saleor/payment/gateways/stripe/tests/test_plugin.py

```diff
@@ -364,6 +364,160 @@ def test_process_payment_with_customer_and_future_usage(
     )
 
 
+@pytest.mark.parametrize(
+    "store_payment_method",
+    [StorePaymentMethodEnum.OFF_SESSION, StorePaymentMethodEnum.ON_SESSION],
+)
+@patch("saleor.payment.gateways.stripe.stripe_api.stripe.Customer.create")
+@patch("saleor.payment.gateways.stripe.stripe_api.stripe.PaymentIntent.create")
+def test_process_payment_with_customer_and_new_future_usage(
```

Comment:
Instead of adding `new` to name I would add `depreceated` to the old tests. Currently, when we will remove the old approach we will have to remove `new` for each tests.

Reply:
And the better way would a creating sepearate test file with deprecated word and place them all tests which operate on old approach.

Reply:
There is a `saleor.payment.gateways.stripe.deprecated.tests` directory, maybe I should put all the old approach tests there?

Reply:
There is a whole deprecated plugin, so I would propose not mixing it up as we want to keep tests together for each plugin.